### ISSUE TITLE
[Question on training log] Negative training loss after 3 epochs

### ISSUE THREAD

**Issue by windspirit95** [speechbrain/speechbrain#1170]
Hi,
I have created a custom dataset by modifying the function create_custom_dataset, and have 3 csv file.
For training I comment from line 541 to 553 (Data preparation) in train.py, and run the training without dynamic mixing.
Training batch size is 1 and I apply the signal length limit to 32000 to avoid OOM.
After 3 epochs I realized the training loss is negative (-0.405). Is this normal phenomenon or something wrong in my training process?
Thank you.

**Comment by windspirit95** [comment]
Here is my training loss, so should I stop the training and use the checkpoint from epoch 2? Thanks.

epoch: 1, lr: 1.50e-04 - train si-snr: 2.23 - valid si-snr: -3.86e+00
epoch: 2, lr: 1.50e-04 - train si-snr: 1.66 - valid si-snr: -3.51e+00
epoch: 3, lr: 1.50e-04 - train si-snr: -4.05e-01 - valid si-snr: -5.13e+00
epoch: 4, lr: 1.50e-04 - train si-snr: -6.35e-01 - valid si-snr: -5.74e+00
epoch: 5, lr: 1.50e-04 - train si-snr: -4.63e-01 - valid si-snr: -5.42e+00
epoch: 6, lr: 1.50e-04 - train si-snr: -9.70e-01 - valid si-snr: -5.79e+00
epoch: 7, lr: 1.50e-04 - train si-snr: -1.22e+00 - valid si-snr: -5.91e+00
epoch: 8, lr: 1.50e-04 - train si-snr: -2.15e+00 - valid si-snr: -6.49e+00
epoch: 9, lr: 1.50e-04 - train si-snr: -1.36e+00 - valid si-snr: -6.81e+00
epoch: 10, lr: 1.50e-04 - train si-snr: -2.33e+00 - valid si-snr: -6.71e+00
epoch: 11, lr: 1.50e-04 - train si-snr: -1.85e+00 - valid si-snr: -6.99e+00
epoch: 12, lr: 1.50e-04 - train si-snr: -2.56e+00 - valid si-snr: -7.41e+00
epoch: 13, lr: 1.50e-04 - train si-snr: -2.43e+00 - valid si-snr: -7.32e+00
epoch: 14, lr: 1.50e-04 - train si-snr: -3.02e+00 - valid si-snr: -7.36e+00
epoch: 15, lr: 1.50e-04 - train si-snr: -2.68e+00 - valid si-snr: -7.70e+00
epoch: 16, lr: 1.50e-04 - train si-snr: -2.62e+00 - valid si-snr: -7.63e+00
epoch: 17, lr: 1.50e-04 - train si-snr: -3.18e+00 - valid si-snr: -7.89e+00
epoch: 18, lr: 1.50e-04 - train si-snr: -3.01e+00 - valid si-snr: -7.70e+00
epoch: 19, lr: 1.50e-04 - train si-snr: -2.83e+00 - valid si-snr: -7.86e+00
epoch: 20, lr: 1.50e-04 - train si-snr: -3.41e+00 - valid si-snr: -7.77e+00
epoch: 21, lr: 1.50e-04 - train si-snr: -3.83e+00 - valid si-snr: -7.91e+00
epoch: 22, lr: 1.50e-04 - train si-snr: -3.70e+00 - valid si-snr: -8.01e+00
epoch: 23, lr: 1.50e-04 - train si-snr: -3.75e+00 - valid si-snr: -8.28e+00
epoch: 24, lr: 1.50e-04 - train si-snr: -3.91e+00 - valid si-snr: -8.23e+00
epoch: 25, lr: 1.50e-04 - train si-snr: -3.66e+00 - valid si-snr: -8.11e+00
epoch: 26, lr: 1.50e-04 - train si-snr: -3.98e+00 - valid si-snr: -8.38e+00
epoch: 27, lr: 1.50e-04 - train si-snr: -4.10e+00 - valid si-snr: -8.39e+00
epoch: 28, lr: 1.50e-04 - train si-snr: -3.09e+00 - valid si-snr: -8.50e+00
epoch: 29, lr: 1.50e-04 - train si-snr: -4.17e+00 - valid si-snr: -8.40e+00
epoch: 30, lr: 1.50e-04 - train si-snr: -4.65e+00 - valid si-snr: -8.40e+00
epoch: 31, lr: 1.50e-04 - train si-snr: -4.44e+00 - valid si-snr: -8.59e+00
epoch: 32, lr: 1.50e-04 - train si-snr: -4.18e+00 - valid si-snr: -8.76e+00
epoch: 33, lr: 1.50e-04 - train si-snr: -4.09e+00 - valid si-snr: -8.78e+00
epoch: 34, lr: 1.50e-04 - train si-snr: -4.05e+00 - valid si-snr: -8.86e+00
epoch: 35, lr: 1.50e-04 - train si-snr: -3.58e+00 - valid si-snr: -8.82e+00
epoch: 36, lr: 1.50e-04 - train si-snr: -3.90e+00 - valid si-snr: -9.02e+00
epoch: 37, lr: 1.50e-04 - train si-snr: -4.24e+00 - valid si-snr: -8.92e+00
epoch: 38, lr: 1.50e-04 - train si-snr: -4.26e+00 - valid si-snr: -8.87e+00
epoch: 39, lr: 1.50e-04 - train si-snr: -5.31e+00 - valid si-snr: -9.20e+00
epoch: 40, lr: 1.50e-04 - train si-snr: -4.51e+00 - valid si-snr: -9.00e+00
epoch: 41, lr: 1.50e-04 - train si-snr: -4.34e+00 - valid si-snr: -9.11e+00
epoch: 42, lr: 1.50e-04 - train si-snr: -5.06e+00 - valid si-snr: -9.29e+00
epoch: 43, lr: 1.50e-04 - train si-snr: -5.06e+00 - valid si-snr: -9.04e+00
epoch: 44, lr: 1.50e-04 - train si-snr: -5.28e+00 - valid si-snr: -9.22e+00
epoch: 45, lr: 1.50e-04 - train si-snr: -4.49e+00 - valid si-snr: -9.17e+00
epoch: 46, lr: 1.50e-04 - train si-snr: -5.27e+00 - valid si-snr: -9.21e+00
epoch: 47, lr: 1.50e-04 - train si-snr: -5.32e+00 - valid si-snr: -9.20e+00
epoch: 48, lr: 1.50e-04 - train si-snr: -4.79e+00 - valid si-snr: -9.44e+00
epoch: 49, lr: 1.50e-04 - train si-snr: -4.71e+00 - valid si-snr: -9.23e+00

**Comment by windspirit95** [comment]
Ah I got the notice about negative si-snr as loss in training tutorial. I close this issue :)
